examples/configuring/README.md

```diff
@@ -0,0 +1,109 @@
+Configuring GraphDB
+===
+
+This document provides detailed instructions on how to configure GraphDB, 
+including setting properties, secret properties, and additional configurations. 
+
+It covers various methods to manage configuration options, such as using 
+ConfigMaps and Secrets, and highlights some good practices to ensure secure 
+and efficient setup. Additionally, it explains how to set Java arguments 
+and environment variables for GraphDB.
+
+All GraphDB configuration options can be found 
+[here](https://graphdb.ontotext.com/documentation/10.0/configuring-graphdb.html#configuration-properties).
```

Comment:
`10.0` ???

Reply:
In version 10.7 the documentation link does not keep all configurations, and information for them, in the same place.
instead, they are scattered across several sections, or at least this is the way I see it

Reply:
https://graphdb.ontotext.com/documentation/10.7/directories-and-config-properties.html#general-properties

Reply:
thanks, I focused more on the main sections of the documentation, not the reference one...

Reply:
fixed